packages/bibliography/tests/test_bibliography.json

```diff
@@ -0,0 +1,70 @@
+{
+    "name": "bibliography",
```

Comment:
Nice with all the tests :)

Reply:
Thanks! The tests are far from flawless since we don't set env on tests (maybe we should add support for that to the test runner). This leads to this test not including `[1]` etc, even though it should, since the sources are never cited (this is how Hayagriva returns uncited sources; without counters)